samples/MerQure.Tools.Samples/RetryBusExample/Infra/SampleChannels.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MerQure.Tools.Configurations;
+using System.Collections.Generic;
+
+namespace MerQure.Tools.Samples.RetryBusExample.Infra
+{
+    public static class SampleChannels
+    { 
+        public static Channel MessageSended = new Channel("v1.sample.messagesended");
+
+        public static Channel OtherAction = new Channel("v1.sample.otheraction");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change the visibility of 'OtherAction' or make it 'const' or 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2223)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this field 'private' and encapsulate it in a 'public' property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1104)